ivy/functional/frontends/tensorflow/sets.py

```diff
@@ -0,0 +1,6 @@
+import ivy
+
+def intersection(x, y, validate_indices=False):
+    # Using Ivy's functional API to compute the intersection of sets
+    intersection_set = ivy.sets.intersection(x, y, validate_indices=validate_indices)
```

Comment:
`ivy` doesn't have an attribute `sets`, you'd need to call it as `ivy.intersection` (and probably also add that to the experimental API)

Reply:
I kinda need a hand with how to and where to add an experimental API,,.